configs/mmseg/segmentation_openvino_static-512x512.py

```diff
@@ -0,0 +1,4 @@
+_base_ = ['./segmentation_static.py', '../_base_/backends/openvino.py']
+
+backend_config = dict(
+    model_inputs=[dict(opt_shapes=dict(input=[1, 3, 512, 512]))])
```

Comment:
As far as I know, OpenVINO in `mmdeploy` seems only support dynamic? @SemyonBevzuk

Reply:
well this config is tested and no error happens.

Reply:
Yeah... But just like PPL.NN, the framework itself has no attribute of `static`, shall we provide static configs to users then?

Reply:
Not a big deal if we keep static configs then